File: trail1/두 사람/two-person.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        string[] brr = Console.ReadLine().Split();
        var(a1, a2) = (int.Parse(arr[0]), arr[1]);
        var(b1, b2) = (int.Parse(brr[0]), brr[1]);
        int val = (a1 >= 19 && a2 == "M") || (b1 >= 19 && b2 == "M") ? 1 : 0;
        Console.WriteLine(val);
    }
}

File: trail1/두 사람/two-person.py
A = list(input().split())
B = list(input().split())

A[0] = int(A[0])
B[0] = int(B[0])

val = 1 if (A[0] >= 19 and A[1] == "M" or B[0] >= 19 and B[1] == "M") else 0
print(val)